exporter/solarwindsexporter/config.go

```diff
@@ -68,12 +42,45 @@ type Config struct {
 	QueueSettings exporterhelper.QueueConfig `mapstructure:"sending_queue"`
 	// Timeout configures timeout in the underlying OTLP exporter.
 	Timeout exporterhelper.TimeoutConfig `mapstructure:"timeout,squash"`
```

Comment:
these OTLP settings are kept in the exporter and not added/reused within the extension as it is presumed those won't be needed for the collector liveness metrics?